@re-/assert/src/snapshot.ts

```diff
@@ -175,75 +99,15 @@ export type QueuedUpdate = {
     newArgText: string
     value: any
     baselineName: string | undefined
+    benchFormat: BenchFormat
 }
 const queuedUpdates: QueuedUpdate[] = []
```

Comment:
I realize this is a carryover from my own code, but let's add a comment to this since it's a global variable with some complexity to it. Something like:

```ts
/**
*   Each time we encounter a snapshot that needs to be initialized or updated, we push its context to the global queuedUpdates variable. Then, on process exit, we call writeUpdates which handles updating all of the affected source (for inline snaps) or JSON (for external snaps or bench history) files.
**/
```

Let's also move the on process exist listener to immediately follow this variable so that it's clear where it's used.